Martinh/update supported networks (#383)
wormhole-foundation/wormhole-docs: build/start-building/supported-networks.md

---
title: Supported Networks

> description: Learn about the networks supported by each Wormhole product across mainnet, testnet, and devnet. Readers can click on each of the blockchain logos for more information.

Reviewer: ```suggestion
description: Learn about the networks supported by each Wormhole product across mainnet, testnet, and devnet. 
```
There are no logos on this page that I can see. Is this outdated?

Reviewer 2: This suggestion is too short just FYI

categories: Reference
---

# Supported Networks

> Wormhole supports many blockchains across mainnet, testnet, and devnet. Product support varies by chain, so each table below lists which networks are compatible with a specific Wormhole product.

Reviewer: ```suggestion
Wormhole supports many blockchains across mainnet, testnet, and devnets. You can use these tables to verify if your desired chains are supported by the Wormhole products you plan to include in your integration. 
```

> ## Supported chains by Product

Reviewer: ```suggestion
## Supported Chains by Product
```

Reviewer 2: I'd suggest to change this to Networks - it just looks weird to me to have the H1 as Supported Networks and this as Supported Chains by Product

Author: Networks by Product?

Reviewer 2: Yeah, I like Networks by Product

# Supported Networks

Wormhole supports several blockchains across multiple environments. Since many concepts are shared within each environment, this section is organized by environment, with individual chains on each page.

Product support may vary by chain. Not all blockchains support the same set of Wormhole products.

To plan your integration:

 - Use the [Product Support tables](#supported-blockchains-by-product) to see which blockchains support each product on mainnet and testnet

>  - Use the [Environment tables](#supported-blockchains-by-environment) below to view supported environments and access developer resources

Reviewer 2: I don't think it makes sense to have separate tables like this. Why not just add environment and quick links to the new table? If some of these tables can be automatically generated, they should be. The scripts should be updated

Author: Do you mean to keep the new tables by product but add an environment and quick link to them? If that's the case, there will be repeated links, but I think it's fine. 

Unfortunately, most of them can't be automatically generated.

Reviewer 2: What do you mean repeated links?

Reviewer 2: Yeah, I mean to me it doesn't make sense to have Supported by Products and Suppoted by Environment. The only thing that Supported by Environment provides that the product tables do not are the links and the environment. So, might as well just add these things to the product tables. Otherwise I feel like there's just way to many tables and not a super clear value add.

Whatever can be automatically generated should. It's difficult to maintain tables like these and stay on top of them.

Reviewer 2: Supported by Product is way more useful for users.

Also we'll need similar tables in the actual product sections since the plan is to have the supported networks available in each section. These tables should be moved to snippets.

| World Chain       | ✅      | ✅       | ❌      |
| X Layer           | ✅      | ✅       | ❌      |
| XPLA              | ✅      | ✅       | ❌      |

### CCTP

| Blockchain        | Mainnet | Testnet | Devnet |
|-------------------|------- -|---------|--------|
| Ethereum          | ✅      | ✅       | ❌      |
| Arbitrum          | ✅      | ✅       | ❌      |
| Avalanche         | ✅      | ✅       | ❌      |
| Base              | ✅      | ✅       | ❌      |
| Optimism          | ✅      | ✅       | ❌      |
| Polygon           | ✅      | ❌       | ❌      |

### Settlement

| Blockchain        | Mainnet | Testnet | Devnet |
|-------------------|---------|---------|--------|
| Ethereum          | ✅      | ❌      | ❌      |
| Solana            | ✅      | ✅      | ❌      |
| Arbitrum          | ✅      | ✅      | ❌      |
| Avalanche         | ✅      | ❌      | ❌      |
| Base              | ✅      | ❌      | ❌      |
| Optimism          | ✅      | ✅      | ❌      |
| Polygon           | ✅      | ❌      | ❌      |
| Sui               | ✅      | ❌      | ❌      |
| Ton               | ✅      | ❌      | ❌      |
| Unichain          | ✅      | ❌      | ❌      |

!!!note

>     Settlement availability depends on the implementation. For example, [Mayan Swift](/docs/learn/transfers/settlement/architecture/#mayan-swift){target=\_blank} is mainnet-only, while others like [Liquidity Layer](/docs/learn/transfers/settlement/architecture/#wormhole-liquidity-layer){target=\_blank} support both mainnet and testnet. Refer to each implementation’s documentation for up-to-date deployment details.

Reviewer 2: There's other implementations, so can we drop a link to a page that shows the users all the implementation options?

Author: There's also MCTP, but since it is a very new implementation, there is not much information about it or use cases

Reviewer 2: Ok but what about when additional implementations get added or MCTP continues to be developed and more use cases arise? We gotta future-proof the docs as much as possible
